packages/interbit-cli/scripts/keys.js

```diff
@@ -1,28 +1,23 @@
-const interbit = require('interbit-core')
-const path = require('path')
-const fs = require('fs-extra')
-const { getArg } = require('interbit')
+const { getArg, keys } = require('interbit')
 
-const keys = async () => {
-  const filenameOption = getArg(process.argv, '--filename')
+const run = async () => {
+  const options = {
+    filename: getArg(process.argv, '--filename')
+  }
 
-  if (!filenameOption) {
+  if (!options.filename) {
     console.error(
       '"interbit keys" command must be used with a --filename option'
     )
     process.exit(1)
   }
 
```

Comment:
How does this work with an existing key file of the same name? Does it throw, fail silently or overwrite? I can live with either throw or overwrite, but not fail silently.

Reply:
I've updated the function that the cli calls to [throw an error if a file already exists.](https://github.com/interbit/interbit/pull/368/commits/73cdda785915668ca59b291656c90413027f8758#diff-2a7b83edaf29e04586e02447667b8620R13)

Reply:
Wouldn't want to wipe out your production keys with a mistype ;)